### PR TITLE
fix .is-center and .is-right

### LINKED ISSUE #62: Align of the container title does not work.
`.container.with-title.is-center(.is-right)` does not work.

https://github.com/BcRikko/NES.css/blob/ceb018d8d6d6ad2d5a3c9937380261d915d1e275/scss/elements/containers.scss#L90-L96

### PR DESCRIPTION
**Pull request by evexoio**
Fixes #62

fix .is-center and .is-right; replace align-items with text-align

screen shot 2018-12-03 at 12 07 32 pm

### CODE REVIEW

**Comment by evexoio** [comment]
@BcRikko Hey! Thanks for taking the time to add more clarity. 🙌

I will add your code to the PR. 👍

File: scss/elements/containers.scss [hunk 1]
```diff
@@ -72,6 +72,21 @@
...
+      background-color: $base-color;
+    }
+
+    &::after {
+      border-color: #fff;
+    }
+
+    > .title {
+      color: $background-color;
+      background-color: $base-color !important;
```
Comment (BcRikko) [inline scss/elements/containers.scss]: I think that `!important` is unnecessary. 🙏 
I don't want to use `!important`. 🙇 
```suggestion
      background-color: $base-color;
```

Comment (BcRikko) [inline scss/elements/containers.scss]: I don't want to use `!important` 😢 
How about below code? 🤔 

```scss
  &.with-title {
    &.is-center { /* ... */ }
    &.is-right { /* ... */ }
  }

  // ↓ write `&.is-dark` below `&.with-title`

  &.is-dark {
    &::before {
      background-color: $base-color;
    }
     &::after {
      border-color: #fff;
    }

    &.with-title {        // <--- add. then Errors are resolved (`Expected selector ".container.is-dark > .title" to come before selector ".container.with-title.is-center > .title"`)
       > .title {
         color: $background-color;
         background-color: $base-color;
       }
   }
  }
```

**Pull request merged**